core/bin/zksync_api/src/fee_ticker/mod.rs

```diff
@@ -204,24 +207,25 @@ const CPK_CREATE2_FEE_TYPE: OutputFeeType = OutputFeeType::ChangePubKey(
 const TOKEN_INVALIDATE_CACHE: Duration = Duration::from_secs(5 * 60);
 
 #[must_use]
-pub fn run_updaters(
-    db_pool: ConnectionPool,
-    config: &zksync_config::TickerConfig,
-) -> Vec<JoinHandle<()>> {
+pub fn run_updaters(db_pool: ConnectionPool, config: &ZkSyncConfig) -> Vec<JoinHandle<()>> {
     let cache = (db_pool.clone(), TokenDBCache::new(TOKEN_INVALIDATE_CACHE));
 
-    let watcher = UniswapTokenWatcher::new(config.uniswap_url.clone());
+    let watcher = CoinGeckoTokenWatcher::new(
+        format!("http://127.0.0.1:9975"),
```

Comment:
This is explained on the next line:
> // !! TODO: use config.ticker.coingecko_base_url.clone() once fee ticker and liquidity ticker are ran on the same mock server !!

Which points out (and reminds me at the same time) the reason why I was arguing for combining the dev-tickers. Otherwise, we could add a new config that would be something like `coinbase_liquidity_base_url`, or use the original `uniswap_base_url` at a cost of clarity. It's your call I guess @antomor.

Reply:
WDYT about adding a config param `coingecko_url`?

Reply:
this will be resolved in https://rsklabs.atlassian.net/browse/ROLLUP-182